protobuf/syft_proto/execution/v1/type_wrapper.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package syft_proto.execution.v1;
+option java_package = "org.openmined.syftproto.execution.v1";
+
+message ClassType {
+    /*
+        A type contains the serialization id and string bytes.
+    */
+    int32 id = 1;
+    bytes type = 2;
+}
+
+message NestedTypeWrapper {
+    /*
+    Message to bufferize a nested type. In a nested type we can have:
+        1. A type.
+        2. A list of nested types.
+        3. A tuple of nested types.
+        4. A dict of nested types.
+    */
+
+    // Message defined to simulate the behavior of a generic map.
+    message KeyValue {
+        // bytes to serialize the key, currently only str is supported.
+        bytes key = 1;
```

Comment:
This can probably be a `string` too.

Reply:
Same as above, I'll use this for a generic map implementation (not sure how yet).

Reply:
How generic does this really need to be? If only `string` is supported, let's just make it a `string`.